src/objects/zcl_abapgit_objects_super.clas.abap

```diff
@@ -461,4 +464,31 @@ CLASS zcl_abapgit_objects_super IMPLEMENTATION.
     ENDIF.
 
   ENDMETHOD.
+
+
+  METHOD is_active.
+
+    DATA: messages    TYPE STANDARD TABLE OF sprot_u WITH DEFAULT KEY,
+          e071_tadirs TYPE STANDARD TABLE OF e071 WITH DEFAULT KEY,
+          e071_tadir  TYPE e071.
+
+    e071_tadir-object   = ms_item-obj_type.
+    e071_tadir-obj_name = ms_item-obj_name.
+
```

Comment:
seems the insert to e071_tadirs is missing?

Reply:
Don't ask how my tests WITH INACTIVE objects could have passed...